test/extended/dr/backup_restore.go

```diff
@@ -0,0 +1,289 @@
+			copyToRecoveryNodeCmd := fmt.Sprintf(`scp %s %s %s -r %s/backup core@%s:`, commonOpts, authOpt, proxyOpt, tempDir, recoveryNodeAddress)
+			runCommandAndRetry(copyToRecoveryNodeCmd)
+
+			// Stop etcd static pods on non-recovery masters.
+			for _, master := range masters {
+				// The restore script will stop static pods on the recovery node
+				if master.Name == recoveryNode.Name {
+					continue
+				}
+				// Recovery 4b
+				g.By(fmt.Sprintf("Stopping etcd static pod on node %q", master.Name))
+				manifest := "/etc/kubernetes/manifests/etcd-pod.yaml"
+				// Move only if present to ensure idempotent behavior during debugging.
+				sudoExecOnNodeOrFail(master, fmt.Sprintf("test -f %s && mv -f %s /tmp || true", manifest, manifest))
+
+				// Recovery 4c
+				g.By(fmt.Sprintf("Waiting for etcd static pod to exit on node %q", master.Name))
+				// Look for 'etcd ' (with trailing space) to be missing to
+				// differentiate from pods like etcd-operator.
+				sudoExecOnNodeOrFail(master, "crictl ps | grep 'etcd ' | wc -l | grep -q 0")
+
+				// Recovery 4f
+				g.By(fmt.Sprintf("Moving etcd data directory on node %q", master.Name))
+				// Move only if present to ensure idempotent behavior during debugging.
+				sudoExecOnNodeOrFail(master, "test -d /var/lib/etcd && (rm -rf /tmp/etcd && mv /var/lib/etcd/ /tmp) || true")
+			}
+
+			// Recovery 4d
+			// Trigger stop of kube-apiserver static pods on non-recovery
+			// masters, without waiting, to minimize the test time required for
+			// graceful termination to complete.
+			for _, master := range masters {
+				// The restore script will stop static pods on the recovery node
+				if master.Name == recoveryNode.Name {
+					continue
+				}
+				g.By(fmt.Sprintf("Stopping kube-apiserver static pod on node %q", master.Name))
+				manifest := "/etc/kubernetes/manifests/kube-apiserver-pod.yaml"
+				// Move only if present to ensure idempotent behavior during debugging.
+				sudoExecOnNodeOrFail(master, fmt.Sprintf("test -f %s && mv -f %s /tmp || true", manifest, manifest))
+			}
+
+			// Recovery 4e
+			// Wait for kube-apiserver pods to exit
+			for _, master := range masters {
+				// The restore script will stop static pods on the recovery node
+				if master.Name == recoveryNode.Name {
+					continue
+				}
+				g.By(fmt.Sprintf("Waiting for kube-apiserver static pod to exit on node %q", master.Name))
+				// Look for 'kube-apiserver ' (with trailing space) to be missing
+				// to differentiate from pods like kube-apiserver-operator.
+				sudoExecOnNodeOrFail(master, "crictl ps | grep -q 'kube-apiserver ' | wc -l | grep -q 0")
+			}
+
+			// Recovery 7
+			restoreFromBackup(recoveryNode)
+
+			// Recovery 8
+			for _, master := range masters {
+				restartKubelet(master)
+			}
+
+			// Recovery 9a, 9b
+			waitForAPIServer(oc.AdminKubeClient(), recoveryNode)
+
+			// Recovery 10,11,12
+			forceOperandRedeployment(oc.AdminOperatorClient().OperatorV1())
```

Comment:
This is disappointing to have to perform.  Is there a reason this is not automatic?

Reply:
I'm just replicating the documented procedure. Is it not in keeping with the current state of the code?

cc: @hexfusion

Reply:
As per slack conversation, this requirement is due to the current state of the static pod controller. Changing that is not in scope for this PR.